Geometry/GEMGeometry/src/ME0Chamber.cc

```diff
@@ -78,3 +78,23 @@ const ME0EtaPartition* ME0Chamber::etaPartition(int isl) const {
   }
   return 0;
 }
+
+float ME0Chamber::computeDeltaPhi(const LocalPoint& position, const LocalVector& direction ) const {
```

Comment:
currently the matching is track position local x - segment position local x and track momentum phi - segment momentum phi. This already removes eta measurement...